### ISSUE TITLE
extended error messages when refreshing repositories

### ISSUE THREAD

**Issue by yoshimo** [NeoApplications/Neo-Store#493]
Sometimes when you refresh the package list you get notification toasts about various kinds of problems.
Unfortunately they are not specific enough to find the cause of the problem nor do they tell you what you can do to fix them

Examples are:
Network error,  is the server down, did i put in the wrong url, was it a dns issue that was maybe caused by an overly broad filter on the local resolver, is my internet connection broken at the moment because my provider has an issue, did i forget to turn on mobile data?
Index broken, do i need to reset the cache, do i need to call the repo owner to fix his stuff?
Signature problems, is that a malicious attack? Did the rep owner change stuff, did i put in the wrong fingerprint?

A bit more details and a bit of a guideline for various errors in these notification toasts would help.

**Comment by ascendbeing** [comment]
I submitted a bug report about that. search for lying or lies from me on here. iirc prior to the last 0.9.x update, possibly prior to the 2nd to last one, you got error messages that told you which repo failed to sync. currently on alpha 5 you're told in my case stack wallet for every single error, which I'm not even subscribed to.
you'll lose some nice features like import and export repos but I might go back to it myself or I might just learn to code lol.

**Comment by machiav3lli** [comment]
You may test current 1.0.0-alpha05 (from the community groups and ) and report back if your issue still consists

**Comment by yoshimo** [comment]
Yes it exists, the errors are still too unspecific and it feels like it became worse. Now only one general error message for all repos is shown instead of 4 slightly more defined ones that weren't helpful either

**Comment by machiav3lli** [comment]
Then it would help if you can describe couple of scenarios to reproduce specific expected issues
